tests/phpunit/tests/theme/wpAddGlobalStylesForBlocks.php

```diff
@@ -20,6 +20,7 @@ class Tests_Theme_WpAddGlobalStylesForBlocks extends WP_Theme_UnitTestCase {
 
 	public function set_up() {
 		parent::set_up();
+		switch_theme( 'default' );
```

Comment:
This and the similar change in the `tear_down()` method are not necessary if #6696 is committed. The reason this fixes the cross-pollution you're seeing is that `wp_clean_theme_json_cache()` gets called on the `switch_theme` action and I think it's better to fix this in the main `WP_Theme_UnitTestCase` method instead of in each test class that extends that method.